components/tickets/PurchaseBottomSheet.tsx

```diff
@@ -0,0 +1,81 @@
+import BottomSheet from '@gorhom/bottom-sheet'
+import { forwardRef, useMemo } from 'react'
+import { View } from 'react-native'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+
+import BottomSheetContent from '@/components/shared/BottomSheetContent'
+import Button from '@/components/shared/Button'
+import Divider from '@/components/shared/Divider'
+import FlexRow from '@/components/shared/FlexRow'
+import Typography from '@/components/shared/Typography'
+import { useTranslation } from '@/hooks/useTranslation'
+import { useGlobalStoreContext } from '@/state/hooks/useGlobalStoreContext'
+
+type Props = object
+
+const PurchaseBottomSheet = forwardRef<BottomSheet, Props>((props, ref) => {
+  const t = useTranslation('PurchaseScreen')
+
+  const { ticketPriceRequest } = useGlobalStoreContext()
+
+  const insets = useSafeAreaInsets()
+  const purchaseButtonContainerHeight = 24 + 12 + 48 + insets.bottom
```

Comment:
I'm not quite sure what exactly those numbers are, please maybe add a comment or extract it to some constant

Reply:
Oh yes, sorry, just fixed "pixels" from Figma, but I think we want to handle this some better way...